media/css/m24/feature.scss

```diff
@@ -60,8 +60,8 @@
     text-wrap-style: balance;
 
     .mzp-c-picto-image {
-        height: 48px;
-        width: 48px;
+        height: 24px;
```

Comment:
**question**: was it also requested to change size on smaller screens?

**issue (potentially blocking)** vertical spacing seems to become bit tight on smaller screen, not sure if that was a design choice

prod
<img width="366" alt="Screenshot 2024-12-17 at 11 14 51 AM" src="https://github.com/user-attachments/assets/06a60d86-1eec-4cd5-89ac-fca6f2625c63" />

pr

<img width="359" alt="Screenshot 2024-12-17 at 11 14 43 AM" src="https://github.com/user-attachments/assets/e0666979-3fdf-4b27-955e-595d568a9b64" />

Reply:
The padding was reduced, so the image size reduction is to make them closer to what we had with the padding icons. This icon size might reduced a bit much. Will increase them slightly and add the space back with extra padding